### PR TITLE
Flutter content handler initializes ICU data when it starts up

### PR DESCRIPTION
**Pull request by vardhan-cr**
Should be submitted alongside https://fuchsia-review.googlesource.com/c/22338/.

This pull request seems to have multiple commits.  Should i be rebasing, or will it get squashed or something?

### CODE REVIEW

**Comment by abarth** [comment]
> Should i be rebasing, or will it get squashed or something?

It will get squashed automatically.

File: content_handler/app.cc [hunk 1]
```diff
@@ -54,13 +55,18 @@ App::App() {
   blink::SetFontProvider(
       context_->ConnectToEnvironmentService<fonts::FontProvider>());
 
+  if (!icu_data::Initialize(context_->environment_services().get())) {
+    FTL_LOG(ERROR) << "Could not initialize ICU data.";
+  }
```
Comment (abarth) [inline content_handler/app.cc]: I'd move this to between lines 49 and 50.

**Comment by abarth** [comment]
LGTM

File: content_handler/app.cc (hunk 1 above)
Comment (vardhan-cr) [inline content_handler/app.cc]: Done.

**Comment by vardhan-cr** [comment]
I don't have merge powers to this repo; could I trouble you to merge this patch along with https://fuchsia-review.googlesource.com/c/22338/ ?

Thanks!

**Pull request merged**